src/Resources/config/client.xml

```diff
@@ -5,10 +5,12 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="nexy_slack.client" class="Nexy\Slack\Client" public="true">
+        <service id="Nexy\Slack\Client" public="true">
             <argument>%nexy_slack.endpoint%</argument>
             <argument>%nexy_slack.config%</argument>
             <argument type="service" id="nexy_slack.http.client" />
         </service>
+
+        <service id="nexy_slack.client" alias="Nexy\Slack\Client"/>
```

Comment:
This really should have been done in the other direction: we typically make the actual service id the snake-case version and then create an alias for the autowiring.